lib/routes/signin-codes.js

```diff
@@ -32,19 +32,19 @@ module.exports = (log, db, customs) => {
         request.validateMetricsContext()
 
         customs.checkIpOnly(request, 'consumeSigninCode')
-          .then(bufferizeSigninCode)
+          .then(hexSigninCode)
           .then(consumeSigninCode)
           .then(reply, reply)
 
-        function bufferizeSigninCode () {
+        function hexSigninCode () {
           let base64 = request.payload.code.replace(/-/g, '+').replace(/_/g, '/')
 
           const padCount = base64.length % 4
           for (let i = 0; i < padCount; ++i) {
             base64 += '='
           }
 
-          return Buffer.from(base64, 'base64')
+          return Buffer.from(base64, 'base64').toString('base64')
```

Comment:
The function name has been changed to `hexSigninCode` but the implementation has been changed to return a base64 string. 😕

Fwiw, I think if this *did* return hex, it would make the associated code in `lib/db.js` simpler...